CONTRIBUTING.md

```diff
@@ -0,0 +1,15 @@
+## Contributing
```

Comment:
Do you think we should we mention on link this file on the pull request template?